### PR TITLE
Changed appropriate instances of 'github' to 'ika'

### PR DESCRIPTION
**Pull request by adambraimbridge**
Solves https://github.com/Financial-Times/github/issues/3
# 🦑🦑🦑🦑🦑🦑🦑🦑🦑🦑🦑
# 🦑🦑🦑🦑🦑🦑🦑🦑🦑🦑🦑
# 🦑🦑🦑🦑🦑🦑🦑🦑🦑🦑🦑
# 🦑🦑🦑🦑🦑🦑🦑🦑🦑🦑🦑
# 🦑🦑🦑🦑🦑🦑🦑🦑🦑🦑🦑
# 🦑🦑🦑🦑🦑🦑🦑🦑🦑🦑🦑

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
+# 🦑 ika
 
-A human-friendly<sup>1</sup> command line tool for the GitHub API.
+## A human-friendly<sup>1</sup> command line tool for the GitHub API.
 
-<sup>1</sup> script-friendly too :)
+<sup>1</sup> It's script-friendly too. 
 
-![image](https://user-images.githubusercontent.com/224547/57020759-3b3c3480-6c22-11e9-8907-565a929d3cd9.png)
+"Ika" (えび) is [Japanese for squid/cuttlefish](https://translate.google.com/#view=home&op=translate&sl=en&tl=ja&text=Squid). Like a squid, the `ika` command line tool is an elongated, fast-swimming cephalopod mollusk with ten arms (technically, eight arms and two long tentacles), typically able to change color.
```
Comment (taktran) [inline README.md]: Should be `いか`

```suggestion
"Ika" (いか) is [Japanese for squid/cuttlefish](https://translate.google.com/#view=home&op=translate&sl=en&tl=ja&text=Squid). Like a squid, the `ika` command line tool is an elongated, fast-swimming cephalopod mollusk with ten arms (technically, eight arms and two long tentacles), typically able to change color.
```

[Finnnnnally get to put my degree in Japanese to good use 😛🇯🇵]

File: bin/ika.js [hunk 2]
```diff
@@ -16,6 +16,7 @@ const baseOptions = flow([
 	commonYargs.withJson(),
 ])
 baseOptions(yargs)
+	.scriptName('ika')
 	/**
 	 * The --version argument only makes sense as an option for the main `github` command.
```
Comment (jenniferemshepherd) [inline bin/ika.js]: does this one need to be 'ika'?

**Comment by jenniferemshepherd** [comment]
Love it!!  🦑

File: README.md (hunk 1 above)
Comment (jenniferemshepherd) [reply README.md]: YOU HAVE A DEGREE IN JAPANESE????

File: bin/ika.js (hunk 2 above)
Comment (jenniferemshepherd) [inline bin/ika.js]: does this one need to be 'ika'? line 21 in new file...

Comment (adambraimbridge) [inline bin/ika.js]:
```suggestion
	 * The --version argument only makes sense as an option for the main `ika` command.
```

**Pull request merged**